scripts/generate_site_sidebar/sidebar_template.md

```diff
@@ -20,5 +20,6 @@
         - [init](reference/live/init/)
         - [preview](reference/live/preview/)
         - [status](reference/live/status/)
+- [Functions Catalog](https://catalog.kpt.dev/ ':crossorgin')
```

Comment:
crossorgin?
is that crossorigin

Also do you have a link to that directive handy?

Reply:
https://docsify.js.org/#/helpers?id=cross-origin-link
The spelling is as used in the documentation.